.travis.yml

```diff
@@ -2,6 +2,9 @@ language: ruby
 rvm:
   - 2.3.0
 script:
+  - bin/rake db:create RAILS_ENV=test
   - bin/rake db:migrate RAILS_ENV=test
   - bundle exec rspec --color --format documentation
   - bundle exec rubocop
+before_script:
+  - cp config/database.yml.travis config/database.yml
```

Comment:
not sure why Travis recommend this overwrite method though: https://docs.travis-ci.com/user/database-setup/#PostgreSQL